private/buf/cmd/buf/command/beta/registry/webhook/webhooklist/webhooklist.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhooklist
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/bufbuild/buf/private/buf/bufcli"
+	"github.com/bufbuild/buf/private/pkg/app/appcmd"
+	"github.com/bufbuild/buf/private/pkg/app/appflag"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	ownerFlagName      = "owner"
+	repositoryFlagName = "repository"
+	remoteFlagName     = "remote"
+)
+
+// NewCommand returns a new Command
+func NewCommand(
+	name string,
+	builder appflag.Builder,
+) *appcmd.Command {
+	flags := newFlags()
+	return &appcmd.Command{
+		Use:   name,
+		Short: "List repository webhooks.",
+		Args:  cobra.ExactArgs(0),
+		Run: builder.NewRunFunc(
+			func(ctx context.Context, container appflag.Container) error {
+				return run(ctx, container, flags)
+			},
+			bufcli.NewErrorInterceptor(),
+		),
+		BindFlags: flags.Bind,
+	}
+}
+
+type flags struct {
+	OwnerName      string
+	RepositoryName string
+	Remote         string
+}
+
+func newFlags() *flags {
+	return &flags{}
+}
+
+func (f *flags) Bind(flagSet *pflag.FlagSet) {
+	flagSet.StringVar(
+		&f.OwnerName,
+		ownerFlagName,
+		"",
+		`The owner name of the repository to list webhooks for.`,
+	)
+	_ = cobra.MarkFlagRequired(flagSet, ownerFlagName)
+	flagSet.StringVar(
+		&f.RepositoryName,
+		repositoryFlagName,
+		"",
+		"The repository name to list webhooks for.",
+	)
+	_ = cobra.MarkFlagRequired(flagSet, repositoryFlagName)
+	flagSet.StringVar(
+		&f.Remote,
+		remoteFlagName,
+		"",
+		"The remote where the repository lives.",
```

Comment:
From the commands here, it doesn't appear that a webhook is attached to a repository, what is this language about?

Reply:
We could improve the naming, but currently as it's designed, all webhooks are tied to a repository, and all events (well, just push haha) are repository-events. I think saying "the remote of the repository" or "the remote of the webhook" both serve well in this command.

Reply:
How about:
Create 
> "The remote of the repository the created webhook will belong to."

Delete
> "The remote of the repository the webhook ID belongs to."

List
> "The remote of the owner and repository to list webhooks for."

Reply:
If this is true, why am I not passing in the repository as part of the list command? Or is it just coincidence that right now, webhooks are per-repo?

Reply:
Yes we're passing the repo in the list command: https://github.com/bufbuild/buf/pull/1293/files#diff-8c45a851004c3df42f95975651e72b86910c8355b35ec4e1728dcf68c84ecfaaR102

Reply:
Create:
`buf beta registry webhook create --owner="achiu" --repository="petapis" --callback-url="https://bufbuild.test/buf.alpha.webhook.v1alpha1.EventService/Event" --event="WEBHOOK_EVENT_REPOSITORY_PUSH"`
Delete:
`buf beta registry webhook delete --id="some_id"`
Listing:
`buf beta registry webhook list --owner="achiu" --repository="petapis" --remote bufbuild.test`

Reply:
got it